pennylane_lightning/core/simulators/lightning_gpu/bindings/LGPUBindings_nb.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright 2025 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file LGPUBindings_nb.hpp
+ * Defines lightning.gpu specific operations to export to Python using Nanobind.
+ */
+
+#pragma once
+#include <complex>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <nanobind/nanobind.h>
+#include <nanobind/ndarray.h>
+#include <nanobind/stl/complex.h>
+#include <nanobind/stl/string.h>
+#include <nanobind/stl/vector.h>
+
+#include "StateVectorCudaManaged.hpp"
+#include "TypeList.hpp"
+
+namespace nb = nanobind;
+
+namespace Pennylane::LightningGPU::NanoBindings {
+
+/**
+ * @brief Define StateVector backends for lightning.gpu
+ */
+using StateVectorBackends = Pennylane::Util::TypeList<
+    StateVectorCudaManaged<float>,
+    Pennylane::Util::TypeList<StateVectorCudaManaged<double>, void>>;
+
+/**
+ * @brief Get a controlled matrix and kernel map for a statevector.
+ * @tparam StateVectorT
+ * @tparam PyClass
+ * @param pyclass Pybind11's measurements class to bind methods.
+ */
+template <class StateVectorT, class PyClass>
+void registerBackendClassSpecificBindings(PyClass &pyclass) {}
+
+/**
+ * @brief Register backend specific measurements class functionalities.
+ *
+ * @tparam StateVectorT
+ * @tparam PyClass
+ * @param pyclass Pybind11's measurements class to bind methods.
```

Comment:
```suggestion
 * @param pyclass Nanobind's statevector class to bind methods.
```

I think? Or is this actually a measurements class getting passed here?